apps/app/src/views/routes/Order.tsx

```diff
@@ -436,13 +453,21 @@ export const Order = () => {
       />
       <Page
         kicker="ORDER"
-        header={
-          loading ? (
+        header=<HStack>
```

Comment:
My understanding of jsx/tsx files is that single tags can be thought of as implicitly wrapped, since they’re replaced with function calls on desugaring. My knowledge may be out of date, though.

Reply:
Could be. I thought it only worked for strings. 🤷